### DISCUSSION TITLE
Angular 18 support

### DISCUSSION THREAD

**Discussion by PeterParushev** [BenjaminBrandmeier/angular2-image-gallery#127] [Ideas]
Hello, do you have plans to upgrade to angular 18? Do you accept contributions as I can do the upgrade and open a PR.

**Comment by BenjaminBrandmeier** [comment]
Hey Peter, thanks for reaching out. 

Angular 18 support is live! 🚀

**Comment by PeterParushev** [comment]
Amazing, thanks!
